src/NWCClient.ts

```diff
@@ -127,6 +127,7 @@ export type Nip47Notification =
 
 export type Nip47PayInvoiceRequest = {
   invoice: string;
+  metadata?: unknown;
```

Comment:
instead of unknown couldn't be an empty object or something?

Reply:
@pavanjoshi914 an empty object won't work, it will create issues when you actually try to pass an object with properties set.